src/layer/lines.js

```diff
@@ -12,33 +16,53 @@ class Lines {
         this.cellCache = [];
         this.$offsetCoefficient = 0;
     }
-    
+
+    /**
+     * @param {import("../../ace").Ace.LayerConfig} config
```

Comment:
In cases like this it is worth to add one typedef instead of writing same import 3 times.

Reply:
It depends on what we prefer more - in case of using [typedef's we need 2 clicks to get real type from `d.ts`](https://github.com/ajaxorg/ace/pull/5318#issuecomment-1724986823) . If this is not a problem, we could place alias on top. Actually, I've done it almost everywhere where I met same type more than once.